utils/xToken.ts

```diff
@@ -0,0 +1,64 @@
+import { api, Extrinsic } from "./setup";
+import {
+  AssetSpec,
+  ChainId,
+  ChainSpecs,
+  TRANSFER_INSTRUCTIONS,
+  WEIGHT_IN_SECONDS,
+} from "./ChainSpecs";
+import { BN } from "@polkadot/util";
+import { User } from "./User";
+
+export class XToken {
+  static xcmTransferFee(toChain: ChainId, assetId: AssetSpec): BN {
```

Comment:
Should we move `xTokenTransfer`  from OAK node to this class?

Reply:
naah, not worth it, would have to rector to have 'api' as argument as well
and kind a like it this way, separate the OAK impl just in case